tensorrt_llm/_torch/models/modeling_speculative.py

```
@@ -590,7 +603,14 @@ def apply_eagle3_fc(self, hidden_states: torch.Tensor) -> torch.Tensor:
 
         expected_hidden_size = self.model.hidden_size
         if hidden_states.shape[-1] != expected_hidden_size:
-            if self.model._norm_before_fc:
+            if self.model.fc_norm is not None:
+                chunks = hidden_states.chunk(len(self.model.fc_norm), dim=-1)
+                hidden_states = torch.cat([
+                    norm(chunk)
+                    for norm, chunk in zip(self.model.fc_norm, chunks)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)